samples/server/petstore/rust-server/output/openapi-v3/src/models.rs

```diff
@@ -1989,10 +1993,10 @@ impl std::str::FromStr for ObjectUntypedProps {
 
         // Use the intermediate representation to return the struct
         std::result::Result::Ok(ObjectUntypedProps {
-            required_untyped: intermediate_rep.required_untyped.into_iter().next().ok_or("required_untyped missing in ObjectUntypedProps".to_string())?,
+            required_untyped: std::result::Result::Err("Nullable types not supported in ObjectUntypedProps".to_string())?,
             required_untyped_nullable: std::result::Result::Err("Nullable types not supported in ObjectUntypedProps".to_string())?,
-            not_required_untyped: intermediate_rep.not_required_untyped.into_iter().next(),
-            not_required_untyped_nullable: intermediate_rep.not_required_untyped_nullable.into_iter().next(),
+            not_required_untyped: std::result::Result::Err("Nullable types not supported in ObjectUntypedProps".to_string())?,
+            not_required_untyped_nullable: std::result::Result::Err("Nullable types not supported in ObjectUntypedProps".to_string())?,
```

Comment:
Rust (rust-server): @metaswitch do these rust changes look good to you?
All any-type properties should allow null in whether nullable is set to true or false.
Per the openapi spec authors nullable True adds nullable to other types like string. One can't remove definition with nullable false per their interpretation.
So in these examples we see null being added from this component
```
    ObjectUntypedProps:
      properties:
        required_untyped:
          nullable: false
        required_untyped_nullable:
          nullable: true
        not_required_untyped:
          nullable: false
        not_required_untyped_nullable:
          nullable: false
      required:
      - required_untyped
      - required_untyped_nullable
      type: object
```
in src/test/resources/3_0/rust-server/openapi-v3.yaml
If I don't hear from you then I will customize the property anytype handling in rust-server to eliminate this change and keep it the same as maser.

Reply:
Added updatePropertyForAnyType to this change is reverted